ui/src/reducers/sampleGrid.js

```diff
@@ -42,10 +42,9 @@ function sampleGridReducer(state = INITIAL_STATE, action = {}) {
   // eslint-disable-next-line sonarjs/max-switch-cases
   switch (action.type) {
     case 'SET_QUEUE': {
-      const sampleList = { ...state.sampleList };
       return {
         ...state,
-        sampleList: Object.assign(sampleList, action.sampleList),
+        sampleList: { ...action.sampleList },
```

Comment:
Just checking: you didn't meant to write `{ ...state.sampleList, ...action.sampleList }`?

Reply:
:), thanks always better to be sure. So I do mean "assign" and not "merge" (like previously)